### ISSUE TITLE
Any chance you could containerize?

### ISSUE THREAD

**Issue by Claire** [zhoubear/open-paperless#6]
Docker container for ease of setup, hosting

**Comment by zhoubear** [comment]
Definitely, working on it. Mayan EDMS has a Docker image but it was too complicated for me. So I'm starting my own image but I'm new to Docker too. I'm aiming at a simple solution.

**Comment by mattsmallman** [comment]
+1

**Comment by bspeice** [comment]
Is there an easy way to get a single container image, rather than the compose image? I've had good experience with supervisord in the past, and am happy to get something spun up myself if needed.

**Comment by schemen** [comment]
Single docker container goes kinda against the concept of docker, 1 process per container should be the goal.
